doc/install.rst

```diff
@@ -210,3 +211,29 @@ Ginga.
 
 After installing Anaconda, open the Anaconda Prompt and follow instructions
 under :ref:`install_generic` via ``conda``.
+
+=============
+Running tests
+=============
+
+#. Install the following packages::
+
+    $ pip install pytest pytest-astropy-header
```

Comment:
What about this?
```suggestion
    $ pip install -e .[test]
```

Reply:
Interesting. Didn't know about that one. You tested that?

Reply:
https://github.com/ejeschke/ginga/blob/c399f2e2b3bd38857b7a15804a2430e3d0cd7180/.travis.yml#L23